### PR TITLE
executor: speedup unit test

### PR DESCRIPTION
**Pull request by crazycs520**
### What problem does this PR solve? 
Try to speed up the unit test. 

Split executor unit test.

before
```go
[2019-07-12T12:55:47.219Z] + go test -vet=off -p 5 -timeout 10m -race -count 1 github.com/pingcap/tidb/executor
[2019-07-12T12:58:53.649Z] ok  	github.com/pingcap/tidb/executor	149.013s
```

This PR
```go
[2019-07-12T12:50:21.176Z] + go test -vet=off -p 5 -timeout 10m -race -count 1 github.com/pingcap/tidb/executor
[2019-07-12T12:52:12.579Z] ok  	github.com/pingcap/tidb/executor	104.146s
```

### Check List 

Tests 

 - No code

Code changes

### CODE REVIEW

**Comment by zhouqiang-cl** [comment]
hi @crazycs520 , any update 😯

**Comment by crazycs520** [comment]
@eurekaka @zz-jason PTAL

File: executor/executor_test.go [hunk 1]
```diff
@@ -98,9 +99,11 @@ var _ = Suite(&testOOMSuite{})
 var _ = Suite(&testPointGetSuite{})
 var _ = Suite(&testRecoverTable{})
 var _ = Suite(&testFlushSuite{})
-var _ = Suite(&testShowStatsSuite{})
 
-type testSuite struct {
+type testSuite struct{ *testSuiteChild }
+type testSuiteP1 struct{ *testSuiteChild }
+
+type testSuiteChild struct {
```
Comment (eurekaka) [inline executor/executor_test.go]: Could you please improve the naming here? e.g, `testSuiteChild` -> `baseTestSuite`, and so on?

Comment (crazycs520) [reply executor/executor_test.go]: Done.

**Comment by crazycs520** [comment]
@qw4990 PTAL

**Pull request merged**